llvm/lib/Target/PowerPC/AsmParser/PPCAsmParser.cpp

```diff
@@ -184,6 +187,7 @@ struct PPCOperand : public MCParsedAsmOperand {
 
   struct ImmOp {
     int64_t Val;
+    bool IsMemOp;
```

Comment:
Technically, this should be `IsMemOpBase` as this is only one component of a memory operand.